Dockerfile

```diff
@@ -2,10 +2,10 @@ ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
 
 # Application image
 
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4
 
 COPY lib/AI-Agent.xml /opt/app/
-COPY build/libs/pip-subscription-management.jar /opt/app/
+COPY build/libs/$APP /opt/app/
```

Comment:
Just tried this out - It looks like $APP isn't getting populated from anywhere and therefore the build is copying over build/libs instead.

I believe we will need to add it into the args section of the docker compose + add an 'ARG'  line in the Dockerfile to get it to be picked up

Reply:
Following Re-Review: Variable not being picked up.

I think the 'ARG APP' needs moving to below the 'FROM' line, otherwise it's not being picked up